spec/dummy/db/structure.sql

```diff
@@ -733,6 +869,49 @@ ALTER SEQUENCE module_refs_id_seq OWNED BY module_refs.id;
 
 
 --
+-- Name: module_runs; Type: TABLE; Schema: public; Owner: -; Tablespace: 
+--
+
+CREATE TABLE module_runs (
+    id integer NOT NULL,
+    attempted_at timestamp without time zone,
+    fail_detail text,
+    fail_reason character varying(255),
+    module_detail_id integer,
```

Comment:
I thought we got rid of this because Mdm::Module::Detail is wiped on reboot of msfconsole or prosvc.